h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -561,6 +561,11 @@ static public boolean contains(int[] a, int d) {
     return false;
   }
 
+  public static <T> boolean contains(T[] arr, T target) {
```

Comment:
This method is not used in this PR.

Reply:
it is, in AutoML

Reply:
I see and this is because you changed the type from String to Enum. In that case I think this method is dangerous because it uses `==` instead of `equals`. I would consider this to be a bug.

Reply:
good point, this was created with enum in mind, but should use equals to follow same semantic as List.contains

Reply:
by the way, there's also a bug in the contains(String[]..) version...